### PR TITLE
added the nuclear option

### PR DESCRIPTION
**Pull request by bthaile**
added 'nuke' to really clean augur-node dir.

### CODE REVIEW

**Comment by epheph** [comment]
Clean already kills `build` , but it doens't hurt. looks good to me

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "test": "mocha test",
     "lint": "tslint -c tslint.json -p . ./src/*.ts ./src/**/*.ts ./src/**/**/*.ts ./src/**/**/**/*.ts && eslint test",
     "build": "tsc",
+    "nuke": "rm -rf node_modules; rm -rf build && npm run clean",
```
Comment (tinybike) [inline package.json]: Suggest also `rm -f package-lock.json; rm -f yarn-lock.json` just to be sure...

Comment (bthaile) [reply package.json]: updated to add the super nuclear option, removing lock files

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "test": "mocha test",
     "lint": "tslint -c tslint.json -p . ./src/*.ts ./src/**/*.ts ./src/**/**/*.ts ./src/**/**/**/*.ts && eslint test",
     "build": "tsc",
+    "nuke": "rm -f package-lock.json; rm -f yarn-local.json; rm -rf node_modules; rm -rf build && npm run clean",
```
Comment (tinybike) [inline package.json]: Should be `rm -f yarn-lock.json` I think (not `yarn-local.json`)

**Comment by justinbarry** [comment]
It is yarn.lock

**Comment by tinybike** [comment]
LGTM

**Comment by tinybike** [comment]
Oh haha, it's yarn.lock... Whew.

**Pull request merged**